dags/genie-nf-validate.py

```diff
@@ -13,8 +13,7 @@
     "pipeline": Param("Sage-Bionetworks-Workflows/nf-genie", type="string"),
     "revision": Param("main", type="string"),
     "profile": Param("aws_prod", type="string"),
-    "only_validate": Param("true", type="string"),
-    "production": Param("true", type="string"),
+    "process_type": Param("only_validate", type="string"),
     "release": Param("13.3-consortium", type="string"),
     "work_dir": Param("s3://genie-bpc-project-tower-scratch/1days", type="string"),
```

Comment:
For some reason, we are also using the spot instance as the default compute env?  can we use on-demand for robustness?